prow/cluster/starter.yaml

```diff
@@ -661,3 +661,15 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: "tide"
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: client-binding
+subjects:
+- kind: User
+  name: client
+roleRef:
+  kind: ClusterRole
+  name: "cluster-admin"
```

Comment:
Instead can we start with documenting what this is doing on https://github.com/kubernetes/test-infra/blob/master/prow/cmd/mkbuild-cluster/README.md?

AKA link the issue you found that helped figure this out. In other words if you get "failed to validate credentials" errors running mkbuild-cluster, you'll need to grant the user (probably named "client") access to pods in that cluster.

I don't actually think we need this object in starter.yaml. The starter.yaml is for a simpler config that schedules jobs in-cluster.

Any chance you'd be able to make that change?

Reply:
Reverted this changes in starter.yaml and adding the instructions in the mkbuild-cluster's README file to give user 'client' access to pods in the cluster.